components/fxa-client/ios/FxAClient/FxAccountManager.swift

```diff
@@ -233,6 +233,20 @@ open class FxAccountManager {
         }
     }
 
+    /// Get the pairing URL to navigate to on the Auth side (typically a computer).
+    public func getPairingAuthorityUrl(
```

Comment:
For consistency with the sibling methods in this file, please use `URL` rather than `Url` in the name here.